README.md

```diff
@@ -1,77 +1,8 @@
-# cytube-docker
+# sync-docker-compose
+Docker compose for sync by calzoneman
 
-Docker image for CyTube
+https://github.com/calzoneman/sync
 
-*This repository is currently very alpha and a work in progress*
+Clone the repo, edit the config file with your information. Make sure the docker-compose file matches your config where needed (port numbers, passwords, certs)
```

Comment:
Could you add a basic getting started section like the previous README had, with example commands for building the image and starting it with docker-compose?